static/sass/_pattern_navigation.scss

```diff
@@ -15,6 +15,18 @@
     font-weight: 400;
     margin-bottom: 0;
 
+    &:target {
+      @media (max-width: $breakpoint-medium) {
+        .p-navigation__nav {
+          background-color: $color-navigation-background;
+        }
+
+        .p-navigation__row {
+          background-color: #fff;
```

Comment:
I know it's unlikely to change but could you please use the Sass colour variable `$color-x-light`. So it is easier to track usage and maintain.
https://docs.vanillaframework.io/en/settings/color-settings

Reply:
I made the change. Sorry about that. Still learning. 😃